chrome_extension/cardboard/manifest.json

```diff
@@ -16,8 +16,8 @@
 	"cookies"
   ],
   "host_permissions": [
-	"0.0.0.0:*",
-	"http://localhost:*/*",
+	"http://*:*/*",

```

Comment:
This feels kind of sketchy, was this just to include 127.0.0.1 or was there more? I guess it's ok as a temporary hack if we need to

Reply:
Also, would this potentially get flagged during the Chrome Extension store review?

Reply:
@rgossiaux, didn't you say you use 0.0.0.0? 

@akirabaruah I actually already included this in the most recent extension upload and it was accepted. However, maybe this is why it took longer.

https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns has Chrome's suggestions. We can include just `http://localhost/*`, `http://127.0.0.1/*`, and `http://0.0.0.0/*`. Their documentation also suggests:

```
You can also use http://*:*/* to match localhost, IP addresses, and any port.
```

Reply:
Ah thanks for the clarifications. SGTM.

I guess we want the `*`s because we want to run this extension on any website, i.e. arbitrary puzzle pages?

Reply:
No. I'm not authorizing using OAuth. I'm instead reading the csrftoken from the website's cookie. This is the list of URLs whose cookies I can read from. The asterisks are needed for folks various ports if you want to test it locally. 

The access to read any website's title and URL is from the permissions's activeTab

Reply:
> @rgossiaux, didn't you say you use 0.0.0.0?

Yeah I do, I was wondering why you changed to the `*:*/*` wildcard pattern since it looked like the only missing local development host was 127.0.0.1. I think `*:*/*` could match any random site, right? I don't think we want to ask people to give us permissions to read their cookies on the whole web if we can avoid it.

Reply:
@rgossiaux I thought `*:*/*` would match anything with a colon but after testing apparently not. I changed it to "http://127.0.0.1/*"  in https://github.com/cardinalitypuzzles/cardboard/pull/737